package.json

```
@@ -28,7 +28,7 @@
   "homepage": "https://github.com/alastria/alastria-identity-example#readme",
   "dependencies": {
     "alastria-identity-JSON-objects": "github:alastria/alastria-identity-JSON-objects#master",
-    "alastria-identity-lib": "git://github.com/alastria/alastria-identity-lib.git#develop"
+    "alastria-identity-lib": "git://github.com/alastria/alastria-identity-lib.git#feature/support-new-contracts"
```

```suggestion
    "alastria-identity-lib": "git://github.com/alastria/alastria-identity-lib.git#develop"
```
https://github.com/alastria/alastria-identity-lib/pull/156#event-4806408296 already merged, so we can point develop again :smile: